project-metrics/metrics_service/metrics/example.py

```diff
@@ -10,8 +10,8 @@
 class IdentityMetric(base.PercentageMetric):
   """A metric which reports the value it is initialized with."""
 
-  def __init__(self, value: float):
-    super(IdentityMetric, self).__init__()
+  def __init__(self, result: models.MetricResult = None, value: float = 0.5):
```

Comment:
Why do these arguments have default values?

Reply:
`result` has a default in the base class, and the `value` default was for convenience in invocation/testing. I'll remove it, doesn't add much.